### PR TITLE
tentative fix for SAN-281

### PR DESCRIPTION
**Pull request by JanKanis**
#ShotGunDebugging

### CODE REVIEW

File: web-app/js/myJobs.js [hunk 1]
```diff
@@ -275,8 +275,11 @@ function checkJobStatus(jobName)	{
 							// Here lies the Venkat rule for standard heatmaps
 							// < 120 seconds, just popup the heatmap.
 							// >= 120 seconds then popup a message to indicate that the heatmap is finished.
-							if (secCount < 120) {  
-								runVisualizerFromSpan(viewerURL, altViewerURL);	
+							if (secCount < 120) {
```
Comment (rnugraha) [inline web-app/js/myJobs.js:278]: Why dont you play around with the resultType == null?

Comment (JanKanis) [inline web-app/js/myJobs.js:278]: Well, this was a quick fix for the problem, so you can apply a real fix when you have time :)

**Comment by kretep** [comment]
I'm going to merge this for now and make a ticket for properly fixing it.

File: web-app/js/myJobs.js (hunk 1 above)
Comment (JanKanis) [inline web-app/js/myJobs.js:278]: resultType is _undefined_ according to the chromium debugger.

**Pull request merged**